_posts/2020-05-07-removing-capybara-webkit.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Removing capybara-webkit"
+date: 2020-04-15 12:00:00
+categories: ["rails", "upgrades"]
+author: arielj
+---
+
+We all know testing is important. We have our unit tests and integration tests to make sure everything is working.
+
+At OmbuLabs, we use [capybara](https://github.com/teamcapybara/capybara) for our integration tests so that we can interact with the app as a real user would.
+
+Capybara by default uses `rack-test` as the driver, but `rack-test` does not support Javascript so to test some things we need a web browser with Javascript capabilities. Having the browser show up while the tests are running is not that practical, now we can use the headless feature on browsers like Chrome or Firefox (even Edge), but a few years ago the available options were mainly PhantomJS or capybara-webkit (through QtWebKit). Those options were deprecated in favor of the major browsers' headless mode.
```

Comment:
capybara-webkit development was suspended because of QtWebKit being deprecated https://github.com/thoughtbot/capybara-webkit#capybara-webkit (in favor of QtWebEngine though, not the major browsers).

I'm not sure if suspended because of qtwebkit being deprecated implies deprecation, so I'll use `development was suspended`

Reply:
@arielj Got it. Then that sentence would sound better like this: 

```
Capybara Webkit is no longer being developed in favor of ... 
```